README.md

```diff
@@ -1,5 +1,59 @@
-## Topcoder Domain - Resource
+## Anticorruption Layer
 
-gRPC service for topcoder anti-corruption layer. This service writes to the legacy database to keep the legacy system happy and available for all legacy systems that interact with it.
+gRPC service for topcoder anti-corruption layer. This service writes changes that happen in v5 to the legacy database to keep the legacy system happy and available for all legacy systems (OR, loaders) that interact with it.
 
-Note that this is still in the initial stages of development and a lot of things might change rapidly. ACL, non-related domains like Challenge, Payment are embedded in this service and will likely be moved to their own services in the future.
+See [domain-challenge](https://github.com/topcoder-platform/domain-challenge) for more information.
+
+# Local Setup
+
+## Dependencies
+
+1. NodeJS 18+
+2. Yarn 1.22+
+3. Typescript 5+
+4. dynamo-access-layer
+5. anticorruption-layer
+
+## Required Environment Variables
+
+The following environment variables are required. Placing a .env file in the root of the project will automatically load these variables.
+
+```env
+GRPC_SERVER_HOST=localhost
+GRPC_SERVER_PORT=40020
+
+GRPC_RDB_SERVER_HOST=localhost
+GRPC_RDB_SERVER_PORT=9090
+
+GRPC_NOSQL_SERVER_HOST=localhost
+GRPC_NOSQL_SERVER_PORT=50051
+
+GRPC_CHALLENGE_DOMAIN_SERVER_HOST=localhost
+GRPC_CHALLENGE_DOMAIN_SERVER_PORT=8888
+
+ENV=local
+```
+
+## Local Development
+
+1. Install dependencies. `anticorruption-layer` uses topcoder-framework which is published in AWS CodeArtifact. To ensure all dependencies are correctly downloaded log into aws codeartifact first
```

Comment:
If the `topcoder-framework` provides the `dynamo-access-layer` and `anticorruption-layer` please state that explicitly somewhere in here.